.github/workflows/automation.yml

```
@@ -0,0 +1,15 @@
+name: 'Automation'
+
+on:
+  push:
+  pull_request_target:
+
+jobs:
+  triage:
+    name: 'Merge conflict labeling'
+    runs-on: ubuntu-latest
```

```suggestion
    runs-on: ubuntu-latest
    if: github.repository == 'jellyfin/jellyfin-web'
```

Forks do not need this workflow, workflows should be disabled by default for forks unless they are useful for contributor forks. Every time I update my web or server fork I instantly get emails from failing builds because of these scripts that I don't need on my repo.

Also same job is disabled in at least the Android TV, Android Mobile and Kotlin SDK repositories.

---

I checked those repos and they indeed have their condition from when we used the old action and ``pull_request`` event.

With ``pull_request_target`` is [not necessary at all](https://docs.github.com/actions/reference/events-that-trigger-workflows#pull_request_target) as the action runs in the context of this repo even in forks. No issues in Vue without the condition.

I'm adding it regardless to stay in the safe side 👍

---

> I'm adding it regardless to stay in the safe side 👍

I think that's the best way to do it with GH Workflows, especially when using custom secrets.